scala/build/Dockerfile.latest

```
@@ -0,0 +1,87 @@
+# Upgrade pip within the virtual environment
+RUN pip install --upgrade pip
+
+# Copy the requirements file into the container
+COPY scala/build/requirements.txt .
+
+# Install dependencies from the requirements file
+RUN pip install -r requirements.txt
+
+
+# Install Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Hadolint

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)